src/github.com/getlantern/fronted/direct.go

```diff
@@ -117,6 +117,24 @@ func (d *direct) NewDirectHttpClient() *http.Client {
 	}
 }
 
+// Do continually retries a given request until it succeeds because some fronting providers
+// will return a 403 for some domains.
+func (d *direct) Do(req *http.Request) (*http.Response, error) {
+	for i := 0; i < 6; i++ {
+		client := d.NewDirectHttpClient()
```

Comment:
We probably could create a single one outside of the loop, but it would just introduce shared state between the requests with maybe saving a few bytes of memory as far as I can tell. What's the advantage of it being outside of the loop?